src/platform.deno.ts

```diff
@@ -64,17 +74,45 @@ export class InputFile {
     constructor(
         file:
             | string
+            | URL
+            | URLLike
             | Uint8Array
             | ReadableStream<Uint8Array>
             | AsyncIterable<Uint8Array>,
         filename?: string,
     ) {
-        this[inputFileData] = file;
+        this.fileData = file;
         if (filename === undefined && typeof file === "string") {
             filename = basename(file);
         }
         this.filename = filename;
     }
+    get [inputFileData]() {
+        if (this.consumed) {
```

Comment:
I looked at this PR again and:
- won't this break `auto-retry`? I mean the plugin won't be able to actually retry the request, because InputFile will be always consumed. 
- I don't see `this.consumed = true` anywhere

Reply:
> won't this break `auto-retry`?

No, because the `consumed` flag is only meant to be set when iterators or other non-reusable data sources are used.

> I don't see `this.consumed = true` anywhere

True. This was fixed in [`23dcf2a` (#77)](https://github.com/grammyjs/grammY/pull/77/commits/23dcf2affda5934e0d93bf2e055308a259612ea3) but it looks like some merge has reverted the change. There is a lot going on in the same files right now, this get a little difficult. Rest assured that it will work after #77.

Reply:
I think the problem was that we should not have merged this before #77, as I stated in the original description.

Reply:
Doesn't GitHub have something to block merges before something else is merged?

Reply:
"PR dependencies" or something like that. I'm not a heavy user of GitHub, so I don't really know